packages/core/src/Actor.ts

```diff
@@ -20,7 +20,7 @@ export interface Actor<
   toJSON: () => {
     id: string;
   };
-  meta?: InvokeDefinition<TContext, TEvent>;
+  meta?: InvokeDefinition<TContext, TEvent, any>;
```

Comment:
Could `InvokeDefinition` default the 3rd param to `any`?

Reply:
I've added the same default as in other places (`{type: string; [key: string]: any}`) 👍